integration_tests/features/Reorgs.feature

```diff
@@ -28,6 +28,34 @@ Feature: Reorgs
     And node B is at height 10
     And node C is at height 10
 
+
+  @critical
+  Scenario: Simple reorg with burned output
+        # Chain 1
+        #     Note: Use more than 1 base node to speed up the test
+    Given I have a seed node SEED_B
+    And I have a base node B connected to seed SEED_B
+    And I have wallet WB connected to base node B
+    And I have mining node BM connected to base node B and wallet WB
+    And mining node BM mines 15 blocks with min difficulty 1 and max difficulty 50
```

Comment:
```suggestion
    And mining node BM mines 10 blocks with min difficulty 1 and max difficulty 1

```

Reply:
These values don't need to be this high. You can also limit the difficulty to 1 per block with the max_difficulty, and then create a longer chain